### DISCUSSION TITLE
Installation of repository

### DISCUSSION THREAD

**Discussion by Remek2GO** [larics/icuas24_competition#23] [General]
Hello,
I am trying to run file install_and_setup_workspace.sh and get an error. Any ideas?
![Przechwytywanie](https://github.com/larics/icuas23_competition/assets/101603785/ff351e85-a4f4-4ceb-82f2-04c25806be71)

**Comment by fpetric** [comment]
The native installation of `uav_ros_simulation` is no longer available due to restricted permissions to certain repositories.
The preferred and easiest method is to simply build the Docker (in icuas24_competition repository) by running:
`
./docker_build.sh`

**Reply by Remek2GO** [reply]
Thank you for information
